src/Autoload.php

```diff
@@ -183,5 +183,14 @@ public static function load($class_name) {
 
 			return false;
 		}
+
+		/**
+		 * Get the array of deprecated Requests 1.x classes mapped to their equivalent Requests 2.x implementation.
+		 *
+		 * @return array
+		 */
+		public static function get_deprecated_classes() {
+			return self::$deprecated_classes;
+		}
```

Comment:
This is fine for me, but I was wondering whether it might be easier here to just turn the `private static $deprecated_classes` into a (public by default) `const DEPRECATED_CLASSES` instead. This would remove the need for an additional method.

Reply:
I seem to remember we discussed that before and it would cause a PHP 5.6 error as we're check against the array with `isset( $array['key'] )` and that syntax is not supported for PHP 5.6 with class constants.

So, having the method is the better solution. It will not prevent us from changing the property to a constant once the minimum supported PHP version goes up to PHP 7.0 (or higher), but will also not necessitate a less efficient/slower implementation of the autoloader.